### PR TITLE
Solve next major related to the API REST

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -64,7 +63,6 @@
         "friendsofsymfony/rest-bundle": "^2.1 || ^3.0",
         "jms/serializer-bundle": "^2.0 || ^3.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.1.1",
-        "nelmio/api-doc-bundle": "^2.4",
```
Comment (jordisala1991) [inline composer.json:67]: you can remove fos rest , jms serializer and nelmio too. Keep in mind the suggest section too.

Comment (VincentLanglet) [reply composer.json:67]: done

**Comment by jordisala1991** [comment]
Did you remove the resources serializer folder? I cant see it on the Pr

**Comment by VincentLanglet** [comment]
> Did you remove the resources serializer folder? I cant see it on the Pr

I didn't. It's removed now

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Nice cleanup

**Comment by jordisala1991** [comment]
You might need to rebase

**Comment by VincentLanglet** [comment]
> You might need to rebase

The branch is already up to date :/

Seems like the branch was not updated by the dev-kit https://github.com/sonata-project/SonataPageBundle/blob/4.x/.github/workflows/test.yaml#L29 @jordisala1991

**Comment by jordisala1991** [comment]
Maybe there is a timeout? we would need to check the cron.

Edit: We hit some rate limit, next run will update it.

**Comment by VincentLanglet** [comment]
Test are ok now :)

**Review by core23 — APPROVED** [review]

**Pull request merged**